examples/mongodbatlas_stream_privatelink_endpoint/confluent_serverless/main.tf

```diff
@@ -0,0 +1,59 @@
+resource "confluent_environment" "staging" {
```

Comment:
Should there be a serverless instance in this example?

Reply:
Serverless products in this case would be for example AWS Lambda, not Confluent serveless products/instance

Reply:
Example purpose is to show how to use & create this resource (mongodbatlas_stream_privatelink_endpoint), in this case no products or services are using it, but this is like all our other examples( for example mongodbatlas_advanced_cluster example creates the cluster, we don't show services/products using the cluster)